tests/unit/Correction.spec.ts

```diff
@@ -59,10 +59,10 @@ describe('Correction - UI', () => {
 
     // init store
     store.state.currentDate = '2020-03-04'
-    store.state.entityType = 'CP'
-    store.state.entityName = 'My Test Entity'
-    store.state.identifier = 'CP1234567'
-    store.state.entityFoundingDate = new Date('1971-05-12T00:00:00-00:00')
+    store.state.business.legalType = 'CP'
+    store.state.business.legalName = 'My Test Entity'
+    store.state.business.identifier = 'CP1234567'
+    store.state.business.foundingDate = '1971-05-12T00:00:00-00:00'
```

Comment:
Just checking, is it okay that the JavaScript date is changing to a date string?

Reply:
In the store (ie, from business object -- from API), it is a string, but to the store users (via getter), it's a JS Date.

Reply:
Remember: the old code converted the string to a Date in App.vue (on fetch/store); now it's done in the getter.